pkg/glusterfs/controllerserver.go

```diff
@@ -0,0 +1,377 @@
+package glusterfs
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	csi "github.com/container-storage-interface/spec/lib/go/csi/v0"
+	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/golang/glog"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+)
+
+const (
+	glusterDescAnn        = "GlusterFS-CSI"
+	glusterDescAnnValue   = "gluster.org/glusterfs-csi"
+	defaultVolumeSizeInMB = 20 * MB // minimum volume size
```

Comment:
Change the default volume size to 1GB(or meaningful default size). 20MB is minimum size validated by Glusterd2, but here this default size will be used when user don't specify the size.

Reply:
@aravindavk  we need to finalize what should be the default size we should provide.
currently in heketi its `1GB` because minimum required size to create volume is `1GB`

Reply:
The constant is renamed and I also use a seperate import to calculate this.